app/notifications/process_notifications.py

```diff
@@ -128,10 +127,7 @@ def send_notification_to_queue(notification, research_mode, queue=None):
 
     if notification.notification_type == SMS_TYPE:
         deliver_task = provider_tasks.deliver_sms
-        if send_to_providers._sms_send_on_pinpoint(
-            notification.notification_type,
-            notification.reply_to_text
-        ):
+        if notification.sends_with_custom_number():
```

Comment:
This method routes SMS to different queues/Celery tasks if they are sent with a dedicated long code number or not.

The logic has been introduced in https://github.com/cds-snc/notification-api/pull/1185 and is the condition is now updated. Previously it was "if sent with Pinpoint go to the throttled queue" now it's "if the SMS is sent with a custom sender, go to the throttled queue"